.gitignore

```diff
@@ -40,3 +40,4 @@ dataactcore/scripts/dropMany.py
 runDynamo.py
 dataactvalidator/config/sf_133*.csv
 runBothTestPackages.py
+dataactcore/dev_config.yml
```

Comment:
Is the plan also to have a `prod_config`, `staging_config` (maybe even `local_config`), etc? If so, should those be in here too?